packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,295 @@
+| onBlur                                                                                             | `({ event, name, value }) => {}`                                          | No       |                                                           | Function called when the input field loses focus                                                                                                                                                                                                                                      |
+| isDisabled                                                                                         | `boolean`                                                                 | No       | `false`                                                   | Marks the input field as disabled. It'll also set `aria-disabled` to true and hence the input field will loose keyboard focus.                                                                                                                                                        |
+| isRequired                                                                                         | `boolean`                                                                 | No       | `false`                                                   | Marks the input field as required. if the errorText is provided then that will be shown if the field is required and not filled, else `${label} is required`                                                                                                                          |
+| necessityIndicator                                                                                 | `optional`, `required`                                                    | No       |                                                           | Indicator to be shown next to the label for emphasizing the necessity of the field. if `optional` is provided then it'll show `${label} (optional)` and if `required` is provided then it'll show `${label} *`                                                                        |
+| icon                                                                                               | `Icon`                                                                    | No       |                                                           | Icon to be displayed at the start of the input field.                                                                                                                                                                                                                                 |
```

Comment:
**Question:** Do we have `<Icon/>` component already, if not, in that case till then are thinking to accept in this prop?

Reply:
ooops sorry 😅